CRISPResso2/CRISPRessoReports/templates/report.html

```diff
@@ -211,7 +211,7 @@ <h5>Prime editing report (all reads aligned to {{report_data.amplicons[0]}})</h5
 			</div>
 			{% endif %}
 
-			{% if report_data['figures']['sgRNA_based_names'][report_data.amplicons[0]] and  report_data['figures']['sgRNA_based_names'][report_data.amplicons[0]]['11b']%}
+			{% if report_data['figures']['sgRNA_based_names'][report_data.amplicons[0]] and '11b' in report_data['figures']['sgRNA_based_names'][report_data.amplicons[0]] and report_data['figures']['sgRNA_based_names'][report_data.amplicons[0]]['11b']|length > 0 %}
```

Comment:
Do we also need to change `report_data['figures']['sgRNA_based_names'][report_data.amplicons[0]]` to `report_data.amplicons[0] in report_data['figures']['sgRNA_based_names']`?